README.md

```diff
@@ -13,6 +13,12 @@
 Supercharge your data science workflow with LineaPy! Just two lines of code captures, analyzes,
 and transforms Python code to extract production data pipelines in minutes.
 
+<p align="center">
+    <b>:point_down: Try It Out! :point_down:</b>

```

Comment:
only worry is if this will look weird on pypi (we dump the readme there which shows up as our library description)